webrender/src/clip.rs

```diff
@@ -169,21 +169,18 @@ impl ClipSources {
         let mut local_outer = Some(LayerRect::max_rect());
         let mut local_inner = local_outer;
         let mut can_calculate_inner_rect = true;
-        let mut can_calculate_outer_rect = true;
+        let mut can_calculate_outer_rect = false;
```

Comment:
Shouldn't we also remove the line touching this variable from `ClipSource::BorderCorner { .. } => {}` arm? Otherwise, it looks inconsistent, e.g.:
  - we start with `false`
  - we meet a rect, flip to `true`
  - we meet a border corner, set it to `false`
  - we meet something else, flip it back to `true`

Reply:
BorderCorner is currently always the last type of clip added. I think that we can even go a bit further here at some point and allow outer rects with border corners, but perhaps it's better to make this change more cautiously?

Reply:
It's just inconsistent with the rest of the logic. The core of this local change is go though "outer rect can be computed unless there is a clip that's not compatible" to "outer rect can't be computed unless there is at least one clip that has it". This is a perfectly reasonable thing to change. But that assignment of "can_calculate_outer_rect = false" in the loop stands away from the core logic.

Reply:
I agree that it's a little bit odd, so I've made the check even less conservative and it looks like things are still working. The latest version of the PR should have this change.